api/v1alpha1/hostdevicenetwork_webhook.go

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
can we move these files into `webhook` directory?

Reply:
This's autogenerated from operator-sdk scaffolding, it's better to not move them.